spec/pdc/v1/release_spec.rb

```diff
@@ -23,6 +23,11 @@
       count = release.where(active: false).count
       count.must_equal 0
     end
+
+    it 'Get release by multi product_version ' do
```

Comment:
Thanks for adding a test! 
Minor: To match the existing convention, it must be `it 'can get releases by multiple product_version' do ` Note `can get`

Reply:
ok, have changed that, would please help to review it again?